scripts/benchmarks_ci.py

```diff
@@ -239,7 +243,7 @@ def __main(args: list) -> int:
                 '**',
                 '*perf-lab-report.json')
 
-            upload.upload(globpath, UPLOAD_CONTAINER, UPLOAD_QUEUE, UPLOAD_TOKEN_VAR, UPLOAD_STORAGE_URI)
+            upload.upload(globpath, upload_container, UPLOAD_QUEUE, UPLOAD_TOKEN_VAR, UPLOAD_STORAGE_URI)
```

Comment:
So currently, this will follow the same path as when we succeed. Which is to grovel the artifact directory for all of the report JSONs, and then upload them one by one. We need to change this to instead, when we have a failure, create a result JSON without any test results, but with the run information, and upload just that one file.

Reply:
How would I create a JSON in Python and where is the run information?